targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/rtl8195a_init.c

```diff
@@ -18,14 +18,14 @@
 #if defined(__CC_ARM) || \
     (defined (__ARMCC_VERSION) && __ARMCC_VERSION >= 6010050)
 
-extern uint8_t Image$$RW_IRAM2$$ZI$$Base[];
-extern uint8_t Image$$RW_IRAM2$$ZI$$Limit[];
+extern uint8_t Image$$RW_IRAM1$$ZI$$Base[];
+extern uint8_t Image$$RW_IRAM1$$ZI$$Limit[];
 extern uint8_t Image$$TCM_OVERLAY$$ZI$$Base[];
 extern uint8_t Image$$TCM_OVERLAY$$ZI$$Limit[];
 extern uint8_t Image$$RW_DRAM2$$ZI$$Base[];
 extern uint8_t Image$$RW_DRAM2$$ZI$$Limit[];
-#define __bss_sram_start__ Image$$RW_IRAM2$$ZI$$Base
-#define __bss_sram_end__   Image$$RW_IRAM2$$ZI$$Limit
+#define __bss_sram_start__ Image$$RW_IRAM1$$ZI$$Base
+#define __bss_sram_end__   Image$$RW_IRAM1$$ZI$$Limit
```

Comment:
This one commit contains few changes:
- moving bss sramd to ram1
- removing some text from sram1
- some changes in the sections regarding lwip, _api and other files

Should this be split into separate commmits , or all are related and what might be beneficial is to describe these changes in the commit message?

I am trying to understand how this fixes gpio and relates to all these changes here

Reply:
Hi, @0xc0170 
The change in "rtl8195a_init.c" is actually from PR #6226. The PR #6226 have not merge yet. This change actually is not related to this gpio toggle slow issue. Once the PR #6226 is merged, I will try to rebase for this fix gpio toggle slow issue.

Reply:
Thanks, that explains it. Please always state dependencies in the description.

Reply:
Hi @0xc0170 , 
The PR #6226 has merged. So I rebased this PR. Please review if there is any issue. Thank you!

Reply:
Please rebase again, I can see the commits from master duplicated here